index.html

```diff
@@ -471,13 +459,13 @@ <h2 class="form-header">Contact me</h2>
                         <p class="form-description">
                             If you have an application you are interested in
                             developing, a
-                            feature that you need built or a project that needs
-                            coding. I’d
+                            feature that you need built or a project that
+                            needs coding. I’d
                             love to help with it!
                         </p>
                     </div>
-                    <!-- Form -->
-                    <form action="https://formspree.io/f/mwkjrezy"
+                    <!-- Form itself -->
+                    <form action="nomgnenoel@gmail.com"
                         method="post">
```

Comment:
- [ ] I have a small suggestion regarding the `width` of your inputs, including the button in your contact form. They appear to be slightly wider than what is shown in the template. Taking a moment to check and adjust their `width` **will help ensure a closer resemblance to the desired design.** 🤓

   ### Your design:
   ![image](https://github.com/noelfoka/portfolio-Microverse-version/assets/107952724/e94219b9-dac4-4eb1-9aff-8ff00e5b138b)

   ### Figma template:
   ![image](https://github.com/noelfoka/portfolio-Microverse-version/assets/107952724/31f49ba8-ce1f-42ed-b523-66b012ebffe0)